lib/KModal.vue

```diff
@@ -182,11 +189,15 @@
         lastFocus: null,
         maxContentHeight: '1000',
         contentHeight: 0,
+        containsKSelect: false,
         scrollShadow: false,
         delayedEnough: false,
       };
     },
     computed: {
+      modalContentHeight() {
+        return this.$refs.content.getBoundingClientRect().height || this.$refs.content.scrollHeight;
```

Comment:
Just to understand, what situation was this `||` supposed to model?

Reply:
when the page initially loads, `this.$refs.content.getBoundingClientRect().height` will very temporarily be `0`, so in that case we will use `this.$refs.content.scrollHeight`, which will be accurate to the real contents of the modal (rather than the length of the dropdown) because the dropdown has not yet been opened.

i'll add in a code comment - please let me know if you think it's a sufficient explanation 🙂 this and several other of my additions were written to counteract all the strange cases i was able to encounter - _without_ this specific check, upon initial load the modal is only rendered tall enough to contain its title, with its contents floating out in space over the grey overlay.

Reply:
Okay, thank you